coap-gateway/config.yaml

```diff
@@ -60,24 +60,6 @@ clients:
         keyFile: "/secrets/private/cert.key"
         certFile: "/secrets/public/cert.crt"
         useSystemCAPool: false
```

Comment:
I don't want to make PR with 2000lines :)

Reply:
Ok, but it will follow after this one, right ?

Reply:
cancelation I want to do as next ... I started this change because sign in returns bad expiresin to the device. Because auth0 doesn't support introspection of token ...